app/views/layouts/main.html.erb

```diff
@@ -52,7 +52,7 @@
 </div>
 <!-- wrap END -->
 
-<%= include_gon %>
+<%= include_gon(init: true) %>
```

Comment:
I added it because I got undefined error in the browser console. After this the problem was gone.